lug/config-bootstraping.yaml

```diff
@@ -0,0 +1,174 @@
+interval: 10 # Interval between pollings
+loglevel: 4 # 1-5
+concurrent_limit: 5
```

Comment:
Change limit to 1 as we'll sync large chunks of file. We do not need such concurrency for bootstrapping.